test/unit/middlewares/error-handler.spec.js

```diff
@@ -206,5 +203,34 @@
 					expect(broker.logger.error).toHaveBeenCalledWith(err);
 				});
 		});
+
+		describe('if the error handler resolves the error', () => {
+			beforeEach(() => {
+				broker.errorHandler = jest.fn(() => Promise.resolve(true));
+			});
+
+			it('should return the response from the error handler', () => {
+				let error = new MoleculerError("Some error");
+				let handler = jest.fn(() => Promise.reject(error));
+
+				const newHandler = mw.localEvent.call(broker, handler, action);
+				const ctx = Context.create(broker, eventEndpoint);
+				ctx.id = "123456";
+				ctx.nodeID = "server-2";
+
+
+				return newHandler(ctx)
+					.then((res) => {
+						expect(res).toBe(true);
+						expect(error.ctx).toBe(ctx);
+						expect(broker.errorHandler).toBeCalledTimes(1);
+						expect(broker.errorHandler).toBeCalledWith(error, {
+							ctx,
+							service: action.service,
```

Comment:
Codacy has a fix for the issue: [Delete `↹`](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141835)
```suggestion
						service: action.service,
```